src/mame/seta/ssv.cpp

```diff
@@ -297,27 +297,27 @@ void ssv_state::lockout_w(offs_t offset, uint16_t data, uint16_t mem_mask)
 //  popmessage("%02X",data & 0xff);
 	if (ACCESSING_BITS_0_7)
 	{
-		machine().bookkeeping().coin_lockout_w(1,~data & 0x01);
-		machine().bookkeeping().coin_lockout_w(0,~data & 0x02);
-		machine().bookkeeping().coin_counter_w(1, data & 0x04);
-		machine().bookkeeping().coin_counter_w(0, data & 0x08);
+		machine().bookkeeping().coin_lockout_w(1, BIT(~data, 0));
+		machine().bookkeeping().coin_lockout_w(0, BIT(~data, 1));
+		machine().bookkeeping().coin_counter_w(1, BIT( data, 2));
+		machine().bookkeeping().coin_counter_w(0, BIT( data, 3));
 //                        data & 0x40?
```

Comment:
Might as well either get rid of this and/or change it to a simpler "Bit 6 is unknown".